platforms-serverless/lambda/index.ts

```diff
@@ -11,7 +11,10 @@ const measure_client = process.hrtime.bigint()
 export async function handler() {
   const fs = require('fs')
   const path = require('path')
-  const files = fs.readdirSync(path.dirname(require.resolve('.prisma/client')))
+  const generatedClientDir = path.dirname(require.resolve('.prisma/client', {
+    paths: [require.resolve('@prisma/client')]
+  }))
+  const files = fs.readdirSync(generatedClientDir)
```

Comment:
Does that still find wherever Prisma CLinet was generated to?

Reply:
Yes, it works for pnpm and is compatible with all package managers